src/main/groovy/de/dkfz/roddy/tools/RoddyIOHelperMethods.groovy

```diff
@@ -362,4 +362,21 @@ class RoddyIOHelperMethods {
     static String printTimingInfo(String info, long t1, long t2) {
         return "Timing " + info + ": " + ((t2 - t1) / 1000000) + " ms"
     }
+
+    /**
+     * Will format a map to a two column table. The width of the first column will be calculated using the length of the
+     * longest element.
+     * @param map
+     * @param cntOfTabs
+     * @param tabSep
+     * @param closureForValue
+     * @return
+     */
+    static List<String> convertMapToTwoColumnsFormattedTable(Map<String, ?> map, int cntOfTabs, String tabSep, Closure closureForValue) {
```

Comment:
What I find problematic about this function is that it produces neither a clean tab-separated nor a clean space-separated and aligned table, but some kind of a mixture. Is this table supposed to be parsed?

Reply:
No it is for console output. Not for parsing! I think there are some libraries which can do this. But this is another dependency and we use it rarely.

Reply:
For console output I would just accept this. Maybe call it something like "...PrettyTable"? Something to express that this is not meant for parsing but for readability only?